release/dist/dist.go

```diff
@@ -315,6 +340,18 @@ func findGo(path string) (string, error) {
 	return toolGo, nil
 }
 
+func findYarn(path string) (string, error) {
```

Comment:
Pretty similar to `findGo` above. I guess they could both call some shared `findTool` function. No strong opinion.

Reply:
done